components/data-set-controls.vue

```diff
@@ -99,6 +130,22 @@
       checkRaster(id) {
         return _.has(this.getDatasets, `${id}.raster`)
       },
+      updateRasterLayer(value) {
+        const datasets = this.getDatasets
+        const raster = datasets[this.getActiveRasterLayer]
+        const option = raster.metadata.layerOptions.find(opt => {
+          return opt.band === this.selectedLayer
+        })
+        this.retrieveRasterLayerByImageId({ imageId: raster.raster.imageId, band: option.band })
+      },
+      optionItems(layers) {
+        // Add value to the array to use in the ui-select
+        const options = layers.map(layer => {
```

Comment:
Semantics:
- band -> value
- layer -> option
- options -> optionItems?

Reply:
Need to take a look at this next sprint.